vm/vm_test.go

```diff
@@ -17,25 +17,30 @@ import (
 	"github.com/ava-labs/hypersdk/chain"
 	"github.com/ava-labs/hypersdk/genesis"
 	"github.com/ava-labs/hypersdk/internal/cache"
-	"github.com/ava-labs/hypersdk/internal/emap"
 	"github.com/ava-labs/hypersdk/internal/mempool"
 	"github.com/ava-labs/hypersdk/internal/trace"
 )
 
 func TestBlockCache(t *testing.T) {
+	t.Skip()
```

Comment:
This is such a bizarre unit test that previously existed... we circumvent vm initialization to test we can fetch blocks from the VM since if it's database doesn't exist, it must be serving requests through cache which feels brittle + too introspect-y. I would be open to just deleting this test instead of skipping it.

Reply:
Will do